15750.md

```diff
@@ -0,0 +1,22 @@
+NIP-15750
+========
+
+Nostr Coinjoin Discovery
+---------------------------------------
+
+`draft` `author:kukks`
+
+## Introduction
+The BTCPay Server Coinjoin plugin offers a coinjoin endpoint discovery feature over Nostr for a coordinator runner and a coinjoin client to easily discover each other.
+
+
+## Specification
+This NIP introduces a NIP16 replaceable kind `15750`, with the following tags:
+* `type`: Signals what kind of coinjoin protocol is being provided. In the case of the BTCPay Server coinjoin plugin, this is `wabisabi`. Other possible options could be `whirlpool`, `bip78`, `joinmarket`, etc.
+* `network`: Signals the bitcoin network the coinjoin protocol is operating on. Always lowercase, can be `mainnet`,`testnet`, `signet`, `regtest`.
+* `endpoint`: A URI of where a user may connect to interact with the coinjoin.
+
+The content of the event COULD be used to describe the coinjoin endpoint, such as fees, round parameters, etc.
```

Comment:
Should these be in a standardized format?

Reply:
They could, but it would up to each `type` to define them